client/app/account/billing/translations/Messages_fr_FR.xml

```diff
@@ -62,6 +62,8 @@
    <translation id="autorenew_service_update_renewal_activation_notice" qtlid="369427">Le renouvellement automatique a été activé sur le(s) service(s) désigné(s) ainsi que votre compte.</translation>
    <translation id="autorenew_service_update_renewal_activation_error" qtlid="369440">Vous avez sélectionné le renouvellement automatique pour au moins un service mais nous n'avons pas pu activé le renouvellement automatique sur votre compte : {0}</translation>
    <translation id="autorenew_guide_help" qtlid="156732">Consultez nos guides en ligne.</translation>
+   <translation id="autorenew_service_debt_before_paying_1">Avant de vous engager sur une nouvelle période de temps, nous vous invitons à d'abord régler votre dette restante.</translation>
```

Comment:
i find this sentence is poorly written (no offense!) maybe replace with something like :
"Avant de vous engager sur une nouvelle durée, nous vous invitons au préalable à vous acquitter de votre dette."